### ISSUE TITLE
Miscellaneous Accessibility Improvements and Completion Verification

### ISSUE THREAD

**Issue by schroerbrian** [ShelterTechSF/sheltertech.org#361]
Ensure all accessibility findings and failures have been accounted for on this https://docs.google.com/spreadsheets/d/1wk4l1ZjWCvxJF0Xjuco8b-lWVws-Vo0aIIRkCYRjbIc/edit#gid=0

This should be the last ticket completed as part of the sheltertech.org accessibility tickets. It is somewhat of a verification ticket where the developer should ensure that nothing was missed in the list of accessibility failures

**Comment by richardxia** [comment]
Closing, as we are in the process of porting the site to a no-code platform, Wix, so that the non-engineering teams at ShelterTech can maintain the corporate site on their own.
